build_system/src/test.rs

```diff
@@ -207,9 +207,6 @@ impl TestArg {
 }
 
 fn build_if_no_backend(env: &Env, args: &TestArg) -> Result<(), String> {
-    if args.use_backend {
```

Comment:
I believe it should have been replaced by:
```rust
if args.backend.is_some()
```
but you know this code better than me.